src/audio.c

```diff
@@ -273,3 +284,85 @@ void record(void) {
 	}
     }
 }
+
+
+/* playing recorded voice keyer messages */
+void *play_thread(void *ptr) {
```

Comment:
With playing in background one can start many parallel playbacks by hitting Fn several times.
This is more a bug, than a feature.

Reply:
Without running it in background we are not able to see the ESC key, so we can not avoid that.
What we can do is reject (simply ignore) new play commands as long as the old one is running.